DataManagementSystem/Service/FTS3ManagerHandler.py

```diff
@@ -31,105 +33,128 @@ class FTS3ManagerHandler( RequestHandler ):
         :param limit: max number of jobs to retrieve
         :param jobAssignmentTag: tag to put in the DB
         :param lastMonitor: jobs monitored earlier than the given date
         :return: json list of FTS3Job
     """
 
-    res = cls.fts3db.getActiveJobs( limit = limit, lastMonitor = lastMonitor, jobAssignmentTag = jobAssignmentTag )
+    res = cls.fts3db.getActiveJobs(
+        limit=limit,
+        lastMonitor=lastMonitor,
+        jobAssignmentTag=jobAssignmentTag)
     if not res['OK']:
       return res
 
     activeJobs = res['Value']
-    activeJobsJSON = json.dumps( activeJobs, cls = FTS3JSONEncoder )
+    activeJobsJSON = json.dumps(activeJobs, cls=FTS3JSONEncoder)
 
-    return S_OK( activeJobsJSON )
+    return S_OK(activeJobsJSON)
 
+  types_updateFileStatus = [dict]
 
-  types_updateFileStatus = [ dict ]
   @classmethod
-  def export_updateFileStatus( cls, fileStatusDict ):
+  def export_updateFileStatus(cls, fileStatusDict):
     """ Update the file ftsStatus and error
        :param fileStatusDict : { fileID : { status , error } }
     """
 
-    return cls.fts3db.updateFileStatus( fileStatusDict )
+    return cls.fts3db.updateFileStatus(fileStatusDict)
+
+  types_updateJobStatus = [dict]
 
-  types_updateJobStatus = [ dict ]
   @classmethod
-  def export_updateJobStatus( cls, jobStatusDict ):
+  def export_updateJobStatus(cls, jobStatusDict):
     """ Update the job Status and error
        :param jobStatusDict : { jobID : { status , error } }
     """
 
-    return cls.fts3db.updateJobStatus( jobStatusDict )
+    return cls.fts3db.updateJobStatus(jobStatusDict)
 
+  types_getNonFinishedOperations = [(long, int), basestring]
 
-  types_getNonFinishedOperations = [( long, int ), basestring]
   @classmethod
-  def export_getNonFinishedOperations( cls, limit, operationAssignmentTag ):
+  def export_getNonFinishedOperations(cls, limit, operationAssignmentTag):
     """ Get all the FTS3Operations that are missing a callback, i.e.
         in 'Processed' state
         :param limit: max number of operations to retrieve
         :return: json list of FTS3Operation
     """
 
-    res = cls.fts3db.getNonFinishedOperations( limit = limit, operationAssignmentTag = operationAssignmentTag )
+    res = cls.fts3db.getNonFinishedOperations(
+        limit=limit, operationAssignmentTag=operationAssignmentTag)
     if not res['OK']:
       return res
 
     nonFinishedOperations = res['Value']
-    nonFinishedOperationsJSON = json.dumps( nonFinishedOperations, cls = FTS3JSONEncoder )
+    nonFinishedOperationsJSON = json.dumps(nonFinishedOperations, cls=FTS3JSONEncoder)
+
+    return S_OK(nonFinishedOperationsJSON)
+
+  types_getOperationsFromRMSOpID = [(long, int)]
+
+  @classmethod
+  def export_getOperationsFromRMSOpID(cls, rmsOpID):
+    """ Get the FTS3Operation associated to a given rmsOpID
+
+        :param rmsOpID: ID of the operation in the RMS
+
+        :return: list of the FTS3Operations JSON matching
```

Comment:
Does this mean "JSON encoded list of FTS3Operations", the trailing "matching" is a bit confusing.

Reply:
yes, I change